doc/man7/migration_guide.pod

```diff
@@ -0,0 +1,1814 @@
+EC_POINT_set_affine_coordinates_GF2m and EC_POINT_set_affine_coordinates_GFp
+
+Applications should use L<EC_POINT_get_affine_coordinates(3)> and
+L<EC_POINT_set_affine_coordinates(3)> instead.
+
+=item EC_POINT_get_Jprojective_coordinates_GFp and EC_POINT_set_Jprojective_coordinates_GFp
+
+These functions are not widely used. Applications should instead use the
+L<EC_POINT_set_affine_coordinates(3)> and L<EC_POINT_get_affine_coordinates(3)>
+functions.
+
+=item EC_POINT_make_affine and EC_POINTs_make_affine
+
+There is no replacement. These functions were not widely used, and OpenSSL
+automatically performs this conversion when needed.
+
+=item EC_POINT_set_compressed_coordinates_GF2m and EC_POINT_set_compressed_coordinates_GFp
+
+Applications should use L<EC_POINT_set_compressed_coordinates(3)> instead.
+
+=item EC_POINTs_mul
+
+This function is not widely used. Applications should instead use the
+L<EC_POINT_mul(3)> function.
+
+=item ENGINE_*
+
+All engine functions are deprecated. An engine should be rewritten as a provider.
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item ERR_func_error_string
+
+=item ERR_get_error_line
+
+=item ERR_get_error_line_data
+
+=item ERR_get_state
+
+=item ERR_load_*
+
+OpenSSL now loads error strings automatically so these functions are not needed.
+
+=item ERR_peek_error_line_data and ERR_peek_last_error_line_data
+
+The new functions are L<ERR_peek_error_func(3)>, L<ERR_peek_last_error_func(3)>,
+L<ERR_peek_error_data(3)>, L<ERR_peek_last_error_data(3)>, L<ERR_get_error_all(3)>,
+L<ERR_peek_error_all(3)> and L<ERR_peek_last_error_all(3)>.
+Applications should use L<ERR_get_error_all(3)>, or pick information
+with ERR_peek functions and finish off with getting the error code by using
+L<ERR_get_error(3)>.
+
+=item EVP_CIPHER_CTX_iv, EVP_CIPHER_CTX_iv_noconst and EVP_CIPHER_CTX_original_iv
+
+Applications should instead use L<EVP_CIPHER_CTX_get_updated_iv(3)>,
+L<EVP_CIPHER_CTX_get_updated_iv(3)> and L<EVP_CIPHER_CTX_get_original_iv(3)>
+respectively.
+See L<EVP_CIPHER_CTX_get_original_iv(3)> for further information.
+
+=item EVP_CIPHER_meth_*, EVP_MD_CTX_set_update_fn, EVP_MD_CTX_update_fn, and EVP_MD_meth_*
+
+See L</Providers are a replacement for engines and low-level method overrides>.
+
+=item EVP_PKEY_CTRL_PKCS7_ENCRYPT, EVP_PKEY_CTRL_PKCS7_DECRYPT,
+EVP_PKEY_CTRL_PKCS7_SIGN, EVP_PKEY_CTRL_CMS_ENCRYPT,
+EVP_PKEY_CTRL_CMS_DECRYPT, and EVP_PKEY_CTRL_CMS_SIGN.
+
+These control operations are not invoked by the OpenSSL library anymore and
+are replaced by direct checks of the key operation against the key type
+when the operation is initialized.
+
+=item EVP_PKEY_CTX_get0_dh_kdf_ukm and EVP_PKEY_CTX_get0_ecdh_kdf_ukm
+
+See the "kdf-ukm" item in L<EVP_KEYEXCH-DH(7)/DH key exchange parameters> and
+L<EVP_KEYEXCH-ECDH(7)/ECDH Key Exchange parameters>.
+These functions are obsolete and should not be required.
```

Comment:
_There are no replacements for these functions._ would be good in here somewhere.  Either as a replacement for this line or in addition to it.

Reply:
The link it points to is the replacement - it shouldnt be needed but it is still there..